src/plugins/intel_gna/tests/unit/gna_export_import_test.cpp

```diff
@@ -154,6 +155,10 @@ class GNAExportImportTest : public ::testing::Test {
             }));
 
         EXPECT_CALL(*mockApi, Gna2InstrumentationConfigAssignToRequestConfig(_, _)).Times(AtLeast(1)).WillRepeatedly(Return(Gna2StatusSuccess));
+
+        InSequence seq;
+        EXPECT_CALL(*mockApi, Gna2DeviceClose(_)).WillOnce(Return(Gna2StatusSuccess));
+        EXPECT_CALL(*mockApi, Gna2MemoryFree(_)).Times(AtLeast(1)).WillRepeatedly(Return(Gna2StatusSuccess));
```

Comment:
Few lines of comment would make it easier for future to understand the purpose of it.